react/features/analytics/AnalyticsEvents.js

```diff
@@ -1,663 +1,474 @@
+ * lib-jitsi-meet's AnalyticsEvents somehow?
+ * @type {string}
  */
-export const SHARED_VIDEO_PAUSED = 'sharedvideo.paused';
+const TYPE_UI = 'ui';
 
 /**
- * Started sharing a YouTube video.
+ * The identifier for the "pinned" action. The local participant has pinned a
+ * participant to remain on large video.
  *
  * @type {String}
  */
-export const SHARED_VIDEO_STARTED = 'sharedvideo.started';
+export const ACTION_PINNED = 'pinned';
 
 /**
- * Confirmed stoppage of the shared YouTube video.
+ * The identifier for the "unpinned" action. The local participant has unpinned
+ * a participant so the participant doesn't remain permanently on local large
+ * video.
  *
  * @type {String}
  */
-export const SHARED_VIDEO_STOPPED = 'sharedvideo.stoped';
+export const ACTION_UNPINNED = 'unpinned';
 
 /**
- * The shared YouTube video had its volume change.
+ * The identifier for the "pressed" action for shortcut events. This action
+ * means that a button was pressed (and not yet released).
  *
  * @type {String}
  */
-export const SHARED_VIDEO_VOLUME_CHANGED = 'sharedvideo.volumechanged';
+export const ACTION_SHORTCUT_PRESSED = 'pressed';
 
 /**
- * Pressed the keyboard shortcut for toggling audio mute.
+ * The identifier for the "released" action for shortcut events. This action
+ * means that a button which was previously pressed was released.
  *
  * @type {String}
  */
-export const SHORTCUT_AUDIO_MUTE_TOGGLED = 'shortcut.audiomute.toggled';
+export const ACTION_SHORTCUT_RELEASED = 'released';
 
 /**
- * Pressed the keyboard shortcut for toggling chat panel display.
+ * The identifier for the "triggered" action for shortcut events. This action
+ * means that a button was pressed, and we don't care about whether it was
+ * released or will be released in the future.
  *
  * @type {String}
  */
-export const SHORTCUT_CHAT_TOGGLED = 'shortcut.chat.toggled';
+export const ACTION_SHORTCUT_TRIGGERED = 'triggered';
 
 /**
- * Toggled the display of the keyboard shortcuts help dialog.
- *
- * @type {String}
+ * The name of the keyboard shortcut or toolbar button for muting audio.
  */
-export const SHORTCUT_HELP = 'shortcut.shortcut.help';
+export const AUDIO_MUTE = 'audio.mute';
 
 /**
- * Pressed the keyboard shortcut for togglgin raise hand status.
- *
- * @type {String}
+ * The name of the keyboard shortcut or toolbar button for muting video.
  */
-export const SHORTCUT_RAISE_HAND_CLICKED = 'shortcut.raisehand.clicked';
+export const VIDEO_MUTE = 'video.mute';
 
 /**
- * Pressed the keyboard shortcut for toggling screenshare.
+ * Creates an event which indicates that a certain action was requested through
+ * the jitsi-meet API.
  *
- * @type {String}
+ * @param {Object} action - The action which was requested through the
+ * jitsi-meet API.
+ * @param {Object} attributes - Attributes to attach to the event.
+ * @returns {Object} The event in a format suitable for sending via
+ *      sendAnalytics.
```

Comment:
nit: The return for these comments has extra indentation.

Reply:
Fixed